src/parser.rs

```diff
@@ -442,10 +447,58 @@ struct SatsPack {
     /// BeiDou: 12 + 3 IGSO + 3 GEO
     /// Galileo: 12
     /// => 58 total Satellites => max 15 rows of data
+    #[cfg_attr(feature = "serde", serde(serialize_with = "serialize_deque"))]
+    #[cfg_attr(feature = "serde", serde(deserialize_with = "deserialize_deque"))]
```

Comment:
```suggestion
    #[cfg_attr(feature = "serde", serde(with = "serde_deque"))]
```

```rust
mod serde_deque {
    fn serialize(...) { ... }
    fn deserialize(...) { ... }
}
```

Reply:
https://serde.rs/field-attrs.html#with

Reply:
Thanks for reviewing this PR @wiktorwieclaw . Your comment has been updated in the PR. We can fit it into 1 like but I have approved the PR as is.